pkg/sloop/webserver/webfiles/sloop_ui.js

```diff
@@ -532,13 +533,13 @@ $(document).ready(function() {
         && ! Date.parse(sessionStorage.getItem('setSelectedEndTime')) < new Date(userDate.getTime() - 3000)){
         userDate = new Date(sessionStorage.getItem('selectedEndTime'));
     }
-    userDate.setMinutes(now.getMinutes() - now.getTimezoneOffset());
+    userDate.setMinutes(userDate.getMinutes() - userDate.getTimezoneOffset());
```

Comment:
does `.getUTCMinutes()` (as seen line 542) work here?

Reply:
Tried `.getUTCMinutes()` here as well but not producing the same behavior. Will talk to Sana to see if the change is needed or not

Reply:
@duke-harlan Updated the pr to address the comment: use one technique to present both user selected time and now to local time. Please review again